### DISCUSSION TITLE
Where to find the latest Windows version ?

### DISCUSSION THREAD

**Discussion by jslegers73** [esnet/iperf#1258] [Q&A]
Where can we find the latest Windows version. I see there is a 3.10.1 version. 
I tried to compile it with Visual Studio but I don't have any coding/visual studio skills.

**Comment by davidBar-On** [comment]
There is a site that distributes new iperf3 executable for Windows. This is not an iperf3 official site it seem to be reliable: https://files.budman.pw/ or https://www.neowin.net/forum/topic/1234695-iperf-3101-windows-build/page/3/ (in both download is done from the same site).
